plugins/lookup/passwordstore.py

```diff
@@ -77,6 +79,24 @@
           - warn
           - empty
           - create
+      lock:
+        description: How to synchronize operations.
+        ini:
+          - section: passwordstore_lookup
+            key: lock
+        type: str
+        default: write
+        choices:
+          - readwrite
+          - write
+          - none
+      locktimeout:
```

Comment:
I think it's easier to read and write with an underscore:
```suggestion
      lock_timeout:
```

Reply:
Nothing else in that module uses underscores, hence the choice, examples:
- passwordstore (instead of password_store)
- returnall (instead of return_all)
- subkey (debatable)
- userpass (user_pass?)
- nosymbols (no_symbols?)

Otherwise, I would have preferred lock_timeout as well.

Reply:
Fair point.